cmd/api/src/model/saved_queries.go

```diff
@@ -0,0 +1,67 @@
+package model
+
+import "fmt"
+
+type SavedQuery struct {
+	UserID string `json:"user_id" gorm:"primaryKey"`
+	Name   string `json:"name" gorm:"primaryKey"`
+	Query  string `json:"query"`
+
+	ID int64 `json:"id"`
+	Basic
+}
```

Comment:
Looks like with this you can have rows with duplicate `id` fields which is probably not what we want

![Screenshot from 2023-09-22 10-04-22](https://github.com/SpecterOps/BloodHound/assets/1761629/7725e200-c498-4529-8ee5-e49ad44e5e7c)